worlds/cv64/__init__.py

```diff
@@ -0,0 +1,331 @@
+            self.carrie_stages = False
+        elif self.options.character_stages.value == self.options.character_stages.option_carrie_only:
+            self.reinhardt_stages = False
+        elif self.options.character_stages.value == self.options.character_stages.option_both:
+            self.branching_stages = True
+
+        self.active_stage_exits = get_normal_stage_exits(self)
+
+        stage_1_blacklist = []
+
+        # Prevent Clock Tower from being Stage 1 if more than 4 S1s are needed to warp out of it.
+        if self.s1s_per_warp > 4 and not self.options.multi_hit_breakables.value:
+            stage_1_blacklist.append(rname.clock_tower)
+
+        # Shuffle the stages if the option is on.
+        if self.options.stage_shuffle:
+            self.active_stage_exits, self.starting_stage, self.active_stage_list = \
+                shuffle_stages(self, stage_1_blacklist)
+        else:
+            self.active_stage_list = [stage for stage in vanilla_stage_order if stage in self.active_stage_exits]
+
+        # Create a list of warps from the active stage list. They are in a random order by default and will never
+        # include the starting stage.
+        self.active_warp_list = generate_warps(self)
+
+    def create_regions(self) -> None:
+        # Create the Menu region.
+        active_regions = {"Menu": Region("Menu", self.player, self.multiworld)}
+
+        # Create every stage region by checking to see if that stage is active.
+        active_regions.update({name: Region(name, self.player, self.multiworld) for name in
+                               get_region_names(self.active_stage_exits)})
+
+        # Add the Renon's shop region if shopsanity is on.
+        if self.options.shopsanity.value:
+            active_regions.update({rname.renon: Region(rname.renon, self.player, self.multiworld)})
+
+        # Add the Dracula's chamber (the end) region.
+        active_regions.update({rname.ck_drac_chamber: Region(rname.ck_drac_chamber, self.player, self.multiworld)})
+
+        # Add the locations to the regions.
+        for reg_name in active_regions:
+            reg_locs = get_region_info(reg_name, "locations")
+            if reg_locs is not None:
+                active_regions[reg_name].add_locations(verify_locations(self.options, reg_locs), CV64Location)
+
+        # Set up the regions correctly.
+        for region in active_regions:
+            self.multiworld.regions.append(active_regions[region])
+
+        # Add the warp Entrances to Menu.
+        active_regions["Menu"].add_exits(get_warp_entrances(self.active_warp_list))
+
+        # Add the Entrances to the stage regions.
+        for reg_name in active_regions:
+            reg_ents = get_region_info(reg_name, "entrances")
+            if reg_ents is not None:
+                active_regions[reg_name].add_exits(verify_entrances(self.options, reg_ents, self.active_stage_exits))
+
+    def create_item(self, name: str, force_classification=None) -> Item:
```

Comment:
please add typing to your args for style consistency.

Reply:
Ooops, looks like I missed a couple from Zunawe's suggestions (another being stage_assert_generate). If you see others let me know.